modules/nf-core/sentieon/bwamem/main.nf

```diff
@@ -0,0 +1,72 @@
+process SENTIEON_BWAMEM {
+    tag "$meta.id"
+    label 'process_high'
+    label 'sentieon'
+
+    secret 'SENTIEON_LICENSE_BASE64'
+
+    // Exit if running this module with -profile conda / -profile mamba
+    if (workflow.profile.tokenize(',').intersect(['conda', 'mamba']).size() >= 1) {
+        exit 1, "Sentieon modules does not support Conda. Please use Docker / Singularity / Podman instead."
+    }
+
+    container 'docker.io/nfcore/sentieon:202112.06'
+
+    input:
+    tuple val(meta), path(reads)
+    tuple val(meta2), path(index)
+    path(fasta)
+    path(fasta_fai)
+
+    output:
+    tuple val(meta), path("*.bam"), path("*.bai"), emit: bam_and_bai
+    path  "versions.yml"          , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args ?: ''
+    def prefix = task.ext.prefix ?: "${meta.id}"
+    def sentieon_auth_mech_base64 = task.ext.sentieon_auth_mech_base64 ?: ''
+    def sentieon_auth_data_base64 = task.ext.sentieon_auth_data_base64 ?: ''
+
+    """
+    export SENTIEON_LICENSE=\$(echo -n "\$SENTIEON_LICENSE_BASE64" | base64 -d)
+
+    if  [ ${sentieon_auth_mech_base64} ] && [ ${sentieon_auth_data_base64} ]; then
+        # If sentieon_auth_mech_base64 and sentieon_auth_data_base64 are non-empty strings, then Sentieon is mostly likely being run with some test-license.
+        export SENTIEON_AUTH_MECH=\$(echo -n "${sentieon_auth_mech_base64}" | base64 -d)
+        export SENTIEON_AUTH_DATA=\$(echo -n "${sentieon_auth_data_base64}" | base64 -d)
+        echo "Decoded and exported Sentieon test-license system environment variables"
+    fi
+
+    INDEX=`find -L ./ -name "*.amb" | sed 's/.amb//'`
+
+    sentieon bwa mem \\
+        $args \\
+        -t $task.cpus \\
+        \$INDEX \\
+        $reads \\
+        | sentieon util sort -r $fasta -t $task.cpus -o ${prefix}.bam --sam2bam -
```

Comment:
Yeah, good point - I guess it could be useful.

However, the module currently only outputs bams and bais (currently, no output channel for cram and output-file extension hardcoded to `bam`). Moreover, the receiving subworkflow expects bam.

How about I make a gitHub-issue for fixing this and then we can just move on for now, as it doesn't seem absolutely essential to the Sentieon-subworkflow?